.claude/agents/openshift-cluster-expert.md

```diff
@@ -0,0 +1,49 @@
+---
+name: ocp-expert
+description: Use this agent when you need to interact with, troubleshoot, or analyze OpenShift or Kubernetes clusters. Examples include: <example>Context: User needs to diagnose why pods are failing to start in their OpenShift cluster. user: 'My pods keep crashing with ImagePullBackOff errors, can you help me figure out what's wrong?' assistant: 'I'll use the openshift-cluster-expert agent to diagnose this pod startup issue.' <commentary>Since the user has a cluster-related problem, use the openshift-cluster-expert agent to investigate the ImagePullBackOff errors and provide troubleshooting guidance.</commentary></example> <example>Context: User wants to understand the current state of their cluster resources. user: 'Can you show me the health status of all my worker nodes and any resource constraints?' assistant: 'Let me use the openshift-cluster-expert agent to analyze your cluster's node health and resource utilization.' <commentary>The user is asking for cluster state analysis, so use the openshift-cluster-expert agent to check node status and resource metrics.</commentary></example> <example>Context: User needs help with complex cluster configuration. user: 'I need to set up a custom monitoring solution that can track specific application metrics across multiple namespaces' assistant: 'I'll use the openshift-cluster-expert agent to design and deploy a custom monitoring solution for your multi-namespace requirements.' <commentary>This requires deep OpenShift/Kubernetes expertise and custom workload deployment, perfect for the openshift-cluster-expert agent.</commentary></example>
```

Comment:
Any objection to keep the description more concise and move the examples to a ## section below the main header. We are trying to set conventions for commands here https://github.com/openshift-eng/ai-helpers/pull/1/files
and will do the same for agents in this line.